DataReader/AppDelegate.swift

```diff
@@ -14,7 +14,10 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     var window: UIWindow?
 
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
-       return true
+        window = UIWindow(frame: UIScreen.main.bounds)
```

Comment:
I know that you wanted to try xib approach but I prefer "less code less bug" aproach